### PR TITLE
Add HMDA Historic Data page

### PR DESCRIPTION
**Pull request by contolini**
Adds a page that will replace our current [HMDA data page](https://www.consumerfinance.gov/data-research/hmda/explore). It includes a form that controls a table linking to CSV files hosted on S3.

## Additions

- `HmdaHistoricDataPage` page type
- `HmdaFilterableForm` form with location, record type and description fields.
- `loan_file_metadata.py` file that provides the record count and file size of every CSV.
- Templates 'n stuff

## Testing

1. Create a `Hmda historic data page` page
1. Optionally add any content to the page
1. Publish the page and you should see something like the screenshot below.

## Todos

- The helper text under the form fields isn't included. Implementing that is proving to be tricky.

## Screenshots

b50ed780-4b26-11e9-95bb-6580e7c7ed67

### CODE REVIEW

File: cfgov/hmda/jinja2/hmda/hmda-explorer-controls.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <div class="content-l_col
+                content-l_col-1">
+        <div class="o-form_group">
+            <fieldset class="o-form_fieldset">
+                <legend class="a-legend">
+                    Records included
+                </legend>
+               {% for radio in form.records %}
+                <div class="m-form-field m-form-field__radio m-form-field__lg-target">
+                    <input class="a-radio" name="{{ radio.data.name }}" type="radio" id="{{ radio.id_for_label }}" value="{{ radio.data.value }}" {% if radio.data.selected %} checked{% endif %}>
```
[the remark this answers is not in the document]
Comment (contolini) [reply cfgov/hmda/jinja2/hmda/hmda-explorer-controls.html]: There are labels for these fields. The bot struggles with jinja variables.

File: cfgov/hmda/jinja2/hmda/hmda-explorer-institutions.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    <tr>
+                        <th class="u-w33pct">Year</th>
+                        <th class="u-w33pct">Transmittal sheet</th>
+                        <th class="u-w33pct">Panel</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td data-label="Year">2017</td>
+                        <td data-label="Transmittal sheet"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2017_transmittal_sheet.csv">CSV</a> (154 KB)</td>
```
Comment (schbetsy) [inline cfgov/hmda/jinja2/hmda/hmda-explorer-institutions.html]: The file sizes listed here (`154 KB`) are placeholders for now. In a future PR, we'll update them with the actual file sizes.

File: cfgov/unprocessed/apps/hmda/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "cfpb-hmda",
+  "description": "The consumerfinance.gov/hmda website.",
+  "homepage": "https://www.consumerfinance.gov/eregulations/",
```
Comment (chosak) [inline cfgov/unprocessed/apps/hmda/package.json]: Is this right?

Comment (contolini) [reply cfgov/unprocessed/apps/hmda/package.json]: No, no it is not.

**Pull request merged**